AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift

```diff
@@ -14,15 +14,15 @@ import Foundation
     public let callKitEnabled: Bool
 
     convenience override public init() {
-        self.init(audioMode: .mono, callKitEnabled: false)
+        self.init(audioMode: .stereo48K, callKitEnabled: false)
```

Comment:
why do we want to default to stero48K?

Reply:
That's Audio Team's decision. @zhemitu-amzn

Reply:
Yes this is a team decision. We'd like to see most SDK users use stereo audio by default, with options available for them to go mono at 48kHz and 16kHz.

Reply:
Could you put **Breaking** on the Changelog since this is big behavioral change. What do you think it would be impact?

Reply:
It won't be a breaking change. The current api will continue to work, just that it will now start Stereo/48KHz audio instead of Mono/16KHz which was the only audio mode before the AudioVideoConfiguration apis are added.
@zhemitu-amzn can you share more details on the impact from Audio side.

Reply:
The AudioMode isn't released to customers yet, if that's what you meant by Breaking change @hokyungh ?

Reply:
I mean it has some behavioral change, which we considered to be breaking and @zhemitu-amzn would you be able to let @zhinang-amazon know the plan on 48KHz on default?

Reply:
Yes will let Zhinan know. This is a breaking change anyway, since a builder has to specify an audio mode to start a meeting now, while in the past there was only one audio mode so no need to choose.

Reply:
The impact for existing users is that they have to modify their code and add .mono16k as the audio mode when they start a conference.

Reply:
@zhemitu-amzn builder need not necessarily specify the Audio Mode when starting a meeting. The old api to start audio session that doesn't accept any parameters will continue to be supported and will internally use the default `AudioVideoConfiguration` with `AudioMode.stereo48K`

Reply:
That's my understand as well.